src/features/groups/components/ContainerList.tsx

```diff
@@ -1,23 +1,30 @@
-import { ContainerIcon } from '@/assets/images/icons';
-import { Card, Icon } from '@/components/ui';
+import { ContainerIcon, PlusIcon } from '@/assets/images/icons';
+import { Button, Card, Icon } from '@/components/ui';
 import { fetchContainerList } from '@/lib/api/container/server';
 import { IContainer } from '@/types/definition';
 
+import Link from 'next/link';
+
 export const ContainerList = async ({ id }: { id: string }) => {
   const containers: IContainer[] = await fetchContainerList(id);
   return (
     <>
-      Container
-      {/* <MenuMeatBall /> */}
-      {/* <MenuPlus /> */}
+      <div className="flex items-center justify-between mb-2 h-12">
+        <h2 className="text-xl">Container</h2>
+        <div className="flex gap-0.5">
+          <Button className="flex justify-center items-center w-12 h-12">
+            <Icon icon={PlusIcon} size={4.5} />
+          </Button>
+        </div>
+      </div>
       {containers.map((container) => (
-        <Card key={container.id}>
-          <div className="flex gap-4 items-center">
+        <Card key={container.id} asChild>
+          <Link href={`/foods?container=${id}`} className="flex gap-4 items-center">
```

Comment:
`group` path parameter should be also entered to align with the specification of FoodList page. But considering that filter works without `group` and we have to pull `groupId` from parent component, I thought we don't need to include group to path parameter.

Reply:
Makes sense to me!

Reply:
@kotaaaa @kanta1207 

Umm... I doubt it.

The displayed filter badges for groups/containers should always match the actual selected options. As per the specification, when selecting a container, the corresponding group is automatically selected.

Therefore, it appears a bit unnatural to me. May I hear your opinions?

*However, we also need to fix the current behavior so that deselecting the group filter automatically deselects the corresponding container filter.

<img width="300" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99148565/3fdc342f-f5a0-4d7d-9ab4-23dc8d77b5a6">

https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99148565/0197bec7-e02f-432f-a288-5d3fdfc750d7

Reply:
> The displayed filter badges for groups/containers should always match the actual selected options. As per the specification, when selecting a container, the corresponding group is automatically selected.

@uskayyyyy 
Your point also makes sense to me. I fixed to always filter both group and container from group single page. Thanks for pointing out!